### DISCUSSION TITLE
Portable as download option

### DISCUSSION THREAD

**Discussion by ethpsantos** [riok/Kreya#182] [Ideas]
I lot of use use machines without access to install things.
Or use the programs inside a PenDrive in outsider computers.

So why not have a portable version?

**Comment by CommonGuy** [comment]
Which operating system are you on? Windows?

**Reply by ethpsantos** [reply]
Yes. Windows.

**Reply by CommonGuy** [reply]
I created the issue https://github.com/riok/Kreya/issues/183 for this. We plan to provide a portable version of Kreya in the near future.

**Reply by ethpsantos** [reply]
Thank you.
